tools/cli/commands.go

```diff
@@ -183,9 +183,7 @@ func RegisterDomain(c *cli.Context) {
 		}
 	}
 	isGlobalDomain := false
-	if !c.IsSet(FlagIsGlobalDomain) {
-		ErrorAndExit(fmt.Sprintf("Option %s must be provided.", FlagIsGlobalDomain), err)
-	} else {
+	if c.IsSet(FlagIsGlobalDomain) {
 		isGlobalDomain, err = strconv.ParseBool(c.String(FlagIsGlobalDomain))
```

Comment:
when the FlagIsGlobalDomain is set, then use this value to set the `isGlobalDomain` pointer